OSVR-Unity/Assets/OSVRUnity/src/Math.cs

```diff
@@ -50,6 +50,37 @@ public static Matrix4x4 ConvertPose(OSVR.ClientKit.Pose3 pose)
             {
                 return Matrix4x4.TRS(Math.ConvertPosition(pose.translation), Math.ConvertOrientation(pose.rotation), Vector3.zero);
             }
+
+            //@todo Rect takes left, top, width, height
+            public static Rect ConvertViewport(OSVR.ClientKit.Viewport viewport)
+            {
+                //Unity expects normalized coordinates, not pixel coordinates
+                //@todo below assumes left and right eyes split the screen in half
+                return new Rect(viewport.Left / (2f*viewport.Width), viewport.Bottom / viewport.Height, viewport.Width/(viewport.Width*2f), 1);
+            }
+
+            //@todo is this correct?
+            public static Matrix4x4 ConvertMatrix(OSVR.ClientKit.Matrix44f matrix)
+            {
+                Matrix4x4 matrix4x4 = new Matrix4x4();
```

Comment:
Hmm.. if I added an indexer to Matrix44f, you could just write for(var i = 0; i < 16; i++) { matrix4x4[i] = matrix[i]; }, if your Matrix44f is already in column major format. Will add that to my todo list.